### ISSUE TITLE
When can I get to use the latest version of hoa?

### ISSUE THREAD

**Issue by taufek** [hoaproject/Protocol#24]
Currently when I install `hoa\project` via composer, it will only use `hoa\protocol` version 1.17.14.
Composer will throw an error. I believe below commit in master suppose to fix it.

https://github.com/hoaproject/Protocol/commit/a832ea9ec9bcbdebf8979d9bae5dfe80e8e1c6f9

Any idea when this fix will be release?

**Comment by Hywan** [comment]
What version do you install exactly?

**Comment by ejunker** [comment]
I have a dependency that requires `hoa/protocol ~1.0` and it is pulling `hoa/protocol 1.17.01.14`. The latest version available shown on https://packagist.org/packages/hoa/protocol is `1.17.01.14` which is behind the latest commits. In particular, as @taufek mentioned the latest version on Packagist is one that defines a global `resolve()` which can cause conflicts with other packages. I see that the latest version of Wrapper.php does not define a global `resolve()` function. Please publish a newer version of this package.

**Comment by Hywan** [comment]
Yes, we are migrating Hoa to the latest PHP version and it takes time. Sorry for the delay.
